wodles/aws/tests/aws_utils.py

```diff
@@ -1,12 +1,11 @@
-# Copyright (C) 2015, Wazuh Inc.
-# Created by Wazuh, Inc. <info@wazuh.com>.
-# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
-
 import os
 import sys
-import copy
 from unittest.mock import patch
 
+import wodles.aws.tests.aws_constants as test_constants
```

Comment:
```suggestion
import aws_constants as test_constants
```

Reply:
Fixed in a2abf24bd3d1b586f03eec8f5371ab9d4b0e1df0.